### ISSUE TITLE
HELO for Mail notifications

### ISSUE THREAD

**Issue by dynek** [crazy-max/diun#165]
### Steps to reproduce this issue

1. Instantiate the docker container with docker provider and mail notifications
### Expected behaviour

Mail should be sent using docker's hostname as HELO or we should be able to specify it

### Actual behaviour

Remote SMTP refuses mail because the HELO value is "localhost":

## Configuration

* Diun version : 4.4.1
* Platform (windows/linux) : Linux
* System info (type `uname -a`) : Linux pve 5.4.24-1-pve #1 SMP PVE 5.4.24-1 (Mon, 09 Mar 2020 12:59:46 +0100) x86_64 GNU/Linux 

### Logs

```
gomail: could not send email 1: 504 5.5.2 <localhost>: Helo command rejected: need fully-qualified hostname","image":"docker.io/library/influxdb:1.8.1-alpine","time":"2020-08-28T19:10:03Z","message":"Mail notification failed"
```

**Comment by crazy-max** [comment]
@dynek Will be done shortly. Thanks for your input.

**Comment by dynek** [comment]
Good, thanks a lot! I was checking it and it doesn't seem more complicated than adding LocalName to gomail.Dialer.

Let me know if you prefer me to create a PR and I'll do it.

**Comment by crazy-max** [comment]
@dynek I'm already on it and like you said yeah `LocalName` is enough ;)

**Comment by crazy-max** [comment]
@dynek Can you try `crazymax/diun:edge` image and let me know if it works? See [doc](https://crazymax.dev/diun/notif/mail/) for the new setting.

**Comment by dynek** [comment]
Oddly enough I do see the environment variable `DIUN_NOTIF_MAIL_LOCALNAME` in the container (docker exec) but I still see "could not send mail" and localhost

**Comment by crazy-max** [comment]
@dynek Forgot the impl. Should be good now.

**Comment by crazy-max** [comment]
@dynek Should be fixed (v4.5.0)

**Comment by dynek** [comment]
Great it's working now 👍
